system/systemd/netdata.service.in

```diff
@@ -10,14 +10,14 @@ Wants=network-online.target nss-lookup.target
 LogNamespace=netdata
 Type=simple
 User=root
+Group=netdata
 RuntimeDirectory=netdata
 RuntimeDirectoryMode=0775
 PIDFile=/run/netdata/netdata.pid
 ExecStart=@sbindir_POST@/netdata -P /run/netdata/netdata.pid -D
 ExecStartPre=/bin/mkdir -p @localstatedir_POST@/cache/netdata
 ExecStartPre=/bin/chown -R @netdata_user_POST@ @localstatedir_POST@/cache/netdata
 ExecStartPre=/bin/mkdir -p /run/netdata
```

Comment:
This also needs to be dropped, systemd will handle it correctly because of `RuntimeDirectory=netdata` above.